avocado/utils/genio.py

```diff
@@ -174,3 +174,16 @@ def write_one_line(filename, line):
     :type line: str
     """
     write_file(filename, line.rstrip('\n') + '\n')
+
+
+class GenIOException(Exception):
```

Comment:
Would you please move this exception class on top of the module and add docstring about it's usage? It significantly improves readability.